apps/console/src/app/app.tsx

```diff
@@ -140,7 +140,7 @@ export function App() {
   return (
     <GTMProvider state={gtmParams}>
       <Routes>
-        <Route path={LOGIN_URL} element={<LoginPage />} />
+        <Route path={LOGIN_URL} element={<PageLogin />} />
```

Comment:
Do you want that we apply this convention for all the project => `PageName` ?

Reply:
Yes I think to automatically generate this we can rename the folder "Pages" to "Page" ?

Reply:
Ah cool, schematics will then create our components with the name PageBlabla? That's a good trick! I like it! Do you want to make the renaming?